tests/rptest/tests/leadership_transfer_test.py

```diff
@@ -108,6 +108,92 @@ def test_self_transfer(self):
                                                     leader)
 
 
+class MultiTopicAutomaticLeadershipBalancingTest(RedpandaTest):
+    topics = (
+        TopicSpec(partition_count=61, replication_factor=3),
+        TopicSpec(partition_count=151, replication_factor=3),
+        TopicSpec(partition_count=263, replication_factor=3),
+    )
+
+    def __init__(self, test_context):
+        extra_rp_conf = dict(leader_balancer_idle_timeout=20000,
+                             leader_balancer_mode="random_hill_climbing")
+
+        super(MultiTopicAutomaticLeadershipBalancingTest,
+              self).__init__(test_context=test_context,
```

Comment:
does reducing `leader_balancer_mute_timeout` help achieve the desired state faster?